platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/CrlCheckerTest.java

```diff
@@ -119,6 +121,48 @@ public void testGetPropertiesFailsBothCertsPass() throws CertificateException {
         assertThat(crlChecker.passesCrlCheck(certs), equalTo(true));
     }
 
+    @Test
+    public void testEnabledCrlFromURLCertFails() throws CertificateException {
+
+        CrlChecker crlChecker = getConfiguredCrlChecker("encryption-crl-none.properties");
+
+        //Get a URL path for the file, this will generate a URL
+        String urlPath = ClassLoader.getSystemResource("crl-revoked.pem")
+                .toString();
+        crlChecker.setCrlLocation(urlPath);
+
+        // First cert should fail
+        String certificateString = getRevokedCert();
+        X509Certificate[] certs = extractX509CertsFromString(certificateString);
+        assertThat(crlChecker.passesCrlCheck(certs), equalTo(false));
+
+        // Second cert should pass
+        certificateString = getUnrevokedCert();
```

Comment:
For these tests that have to conditions, do you think it would be better to split these into two separate tests so if they fail we know exactly which check failed? Or does it make more sense for these two checks to be together?